app/views/shared/_new_form.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="form_pages">
+  <%= form_for @new_task do |n| %>
```

Comment:
For some reason, the naming convention for the form_for is to use the `|f|`. I don't know why! But thats how it is!